src/app/Plans/components/Wizard/SelectVMsForm.tsx

```diff
@@ -275,20 +275,28 @@ const SelectVMsForm: React.FunctionComponent<ISelectVMsFormProps> = ({
           <Pagination {...paginationProps} widgetId="vms-table-pagination-top" />
         </LevelItem>
       </Level>
-      <Table
-        aria-label="VMware VMs table"
-        variant={TableVariant.compact}
-        cells={columns}
-        rows={rows}
-        sortBy={sortBy}
-        onSort={onSort}
-        onCollapse={(event, rowKey, isOpen, rowData) => {
-          toggleVMsExpanded(rowData.meta.vm);
-        }}
-      >
-        <TableHeader />
-        <TableBody />
-      </Table>
+      {filteredItems.length > 0 ? (
+        <Table
+          aria-label="VMware VMs table"
+          variant={TableVariant.compact}
+          cells={columns}
+          rows={rows}
+          sortBy={sortBy}
+          onSort={onSort}
+          onCollapse={(event, rowKey, isOpen, rowData) => {
+            toggleVMsExpanded(rowData.meta.vm);
+          }}
+        >
+          <TableHeader />
+          <TableBody />
+        </Table>
+      ) : (
+        <TableEmptyState
+          titleText="No VMs found"
+          bodyText="No results match your filter. Go back and make a different selection."
```

Comment:
This one is good though.

Reply:
Oh wait actually, we shouldn't say "go back" here either, because the filters are on the same step in this case. I only had that phrasing for the case where you have to actually go back to the Filter VMs wizard step.

Reply:
@gildub ^ (sorry if you were making a commit before I realized my mistake)